pinecone/index_connection_test.go

```diff
@@ -0,0 +1,161 @@
+package pinecone
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"os"
+	"testing"
+)
+
+type IndexConnectionTests struct {
+	suite.Suite
+	host      string
+	apiKey    string
+	idxConn   *IndexConnection
+	namespace string
+	vectorIds []string
+}
+
+// Runs the test suite with `go test`
+func TestIndexConnection(t *testing.T) {
+	apiKey := os.Getenv("API_KEY")
+
+	podTestSuite := new(IndexConnectionTests)
+	podTestSuite.host = os.Getenv("POD_INDEX_HOST")
```

Comment:
It looks like these tests expect the index to already exist. Is that handled somewhere in this repo or is that currently a manual setup step?

Reply:
The indexes have to already exist with the hosts being in .env. I expect this to change when the control plane api is added to the repo.